src/ui.form-input-money/utilFormat.js

```diff
@@ -65,16 +65,11 @@ export default class FormatService {
       rawValuePrefix: false,
     });
 
-    const formattedValue = intlNumber.formatToParts(parseFloat(rawValue) || 0).map((part) => {
-      if (part.type === "group") part.value = thousandsSeparator;
-      if (part.type === "decimal") part.value = decimalSeparator;
-      if (part.type === "fraction") part.value = part.value.padEnd(maxFractionDigits, "0");
+    const formattedValue = intlNumber
+      .format(rawValue)
+      .replaceAll(this.comma, thousandsSeparator)
```

Comment:
replaceAll is es2021 feature, it's better to use `replace` with `/g` flag for better browser compatibility.